index.js

```diff
@@ -228,7 +229,7 @@ function spawnEntity(packet, client) {
 
 function sendActions() {
 	const newStates = {punching: lStates[0] > Date.now(), sprinting: lStates[1], sneak: lStates[2]};
-	if (newStates.punching == lastLState.punching && newStates.sprinting == lastLState.sprinting && newStates.sneak == lastLState.sneak) return;
+	if (newStates.punching == lastLState.punching && newStates.sprinting == lastLState.sprinting && newStates.sneak == lastLState.sneak) {return;}
```

Comment:
```suggestion
	if (newStates.punching == lastLState.punching && newStates.sprinting == lastLState.sprinting && newStates.sneak == lastLState.sneak) return;
```
Bart, say the line again...
> NO MORE BRACKETS!